test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Models/PackageItemListViewModelTests.cs

```diff
@@ -3,28 +3,49 @@
 
 using System;
 using System.Collections.Generic;
-using System.Linq;
-using System.Text;
+using System.IO;
+using System.IO.Packaging;
 using System.Threading.Tasks;
+using System.Windows.Media;
+using System.Windows.Media.Imaging;
 using NuGet.Packaging;
-using NuGet.Protocol;
 using NuGet.Test.Utility;
 using Xunit;
+using Xunit.Abstractions;
 
 namespace NuGet.PackageManagement.UI.Test
 {
     public class PackageItemListViewModelTests : IClassFixture<LocalPackageSearchMetadataFixture>
     {
         private readonly LocalPackageSearchMetadataFixture _testData;
         private readonly PackageItemListViewModel _testInstance;
+        private static readonly ImageSource DefaultPackageIcon;
+        private readonly ITestOutputHelper output;
 
-        public PackageItemListViewModelTests(LocalPackageSearchMetadataFixture testData)
+        public PackageItemListViewModelTests(ITestOutputHelper output, LocalPackageSearchMetadataFixture testData)
         {
             _testData = testData;
             _testInstance = new PackageItemListViewModel()
             {
                 PackageReader = _testData.TestData.PackageReader,
             };
+            this.output = output;
+        }
+
+        static PackageItemListViewModelTests()
+        {
+            // ensure that pack: scheme is registered, otherwise pack: uris won't work in tests.
+            const string scheme = "pack";
+            if (!UriParser.IsKnownScheme(scheme))
+            {
+                // ensure that the pack scheme is registered
+                Assert.Equal(PackUriHelper.UriSchemePack, scheme);
+
+                // ensure that an application exists, so pack application uris work.
+                new System.Windows.Application();
```

Comment:
How does instantiating an `Application` register the "pack" scheme?

Reply:
pack://application is necessary.
PackUriHelper.UriSchemePack enables pack:

new application enables application.

however, several tests on CI are not happy that we have multiple applications around .... so i need to rework this somehow...to not stomp on other tests.

Reply:
Do you literally mean `pack://application` where `application` is significant or you just mean the `pack` scheme?

Maybe you need a fixture?

Reply:
yes, i do mean pack://application -- 
we actually aren't ever really trying to support relative URL loading in a WPF app...and that is to download the default icon.
so, i think i can remove the need for it all together. need to look...

Reply:
changed how this works. tests are happy now.